pages/index.stories.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Home from './index';
+
+storiesOf('Home', module).add('home', () => {
+  return <Home />;
+});
```

Comment:
`storiesOf` API will be deprecated. Could you refer https://github.com/axross/react-holdem and use `addon-docs`-based style instead?